### ISSUE TITLE
ALT/title translation

### ISSUE THREAD

**Issue by jand000** [unclecheese/KickAssets#10]
More a feature request than a bug:
For the same image it is not possible the have individual titles (or ALT text) on each page. Especially for translations this would be important (accessibility and SEO).

**Comment by unclecheese** [comment]
I'm not sure what you mean?

**Comment by jand000** [comment]
Sorry, I try to explain what I mean: For each image on the website the alt attribute is needed ( <img src="1.gif" alt="one" /> ). If the website is in more than one language the alt should be translated. Also the image might be used in a different context on different pages. So you would need a title/alt on page version level. If I got this right KickAssets is only capable of having one title on asset level.
A good example is the ImageGallery module where this title/alt on page/translation level works very well.

**Comment by unclecheese** [comment]
I think you're talking about two different things. KA manages File objects directly, like AssetAdmin. ImageGallery manages DataObjects that have a relation to a file, so that's where all those other fields come from, such as "Caption."

Is this something you could do in AssetAdmin?

---

SilverStripe tips, tutorials, screencasts and more: http://www.leftandmain.com

**Comment by jand000** [comment]
You are absolutely right. It's just that with KA these two things are kind of mixed from UI perspective (which is a good approach, I think). Especially the KA multiple file field could replace the ImageGallery module – if it would be enhanced/mixed with the DataObjects though.
But I'm no programmer; so if that makes no sense: never mind ;)
